ci/Conformance.md

```diff
@@ -0,0 +1,67 @@
+# Conformance results
+
+## To check for the existing nodes
+```
+$ kubectl get nodes
```

Comment:
Don't use the $ sign in the text.  This makes it more difficult to copy and past the commands.